i18n/locales/de/common.json

```diff
@@ -1,238 +1,532 @@
+  "Your session was timed out after 10 minutes of no network activity. You may continue to use certain sections of your Lisk Hub or sign back in to access everything.": "Your session was timed out after 10 minutes of no network activity. You may continue to use certain sections of your Lisk Hub or sign back in to access everything.",
+  "Your signed message": "Deine signierte Nachricht",
+  "Your voice matters": "Deine Stimme zählt.",
+  "You’ll see it in Delegates and it will be confirmed in a matter of minutes.": "You’ll see it in Delegates and it will be confirmed in a matter of minutes.",
+  "from": "von",
+  "or": "oder",
+  "{{deviceModel}} connected! Open the Lisk app on the device": "{{deviceModel}} verbunden! Öffne die Lisk App auf dem Gerät",
+  "{{fee}} {{token}}": "{{fee}} {{token}}",
+  "{{length}} bytes left": "{{length}} verbleibende Bytes",
+  "{{length}} extra characters": "{{length}} verbleibende Zeichen",
+  "{{length}} out of {{maxLength}} characters left": "{{length}} von {{maxLength}} verleibende Zeichen ",
+  "{{passphraseName}} paper wallet": "{{passphraseName}} Papier wallet",
+  "{{token}} Balance": "{{token}} Kontostand",
+  "{{token}} Wallet": "{{token}} Wallet",
+  "{{token}} balance": "{{token}} Kontostand",
+  "Account Balance:          {{balance}} {{token}}": "Account Balance:          {{balance}} {{token}}",
+  "Address already bookmarked": "Address already bookmarked",
+  "Become a delegate summary": "Become a delegate summary",
+  "Bookmarked": "Bookmarked",
+  "DD MMM YY, HH:mm": "DD MMM YY, HH:mm",
+  "DD MMM YYYY": "DD MMM YYYY",
+  "DD MMM YYYY, hh:mm A": "DD MMM YYYY, hh:mm A",
+  "DD.MM.YY": "DD.MM.YY",
+  "Discord": "Discord",
+  "Discreet Mode Disabled": "Der Diskreter Modus ist deaktiviert",
+  "Discreet Mode Enabled": "Der Diskreter Modus ist aktiviert",
+  "Each LSK token is worth one vote.": "Each LSK token is worth one vote.",
+  "English": "Englicsh",
+  "Filter by name...": "Filtern nach Namen...",
+  "German": "Deutsch",
+  "Go to Dashboard": "Gehe zum Dashboard",
+  "Go to Wallet": "Gehe zum Wallet",
+  "Go to confirmation": "Gehe zum Bestätigung",
+  "Go to settings": "Gehe zum Einstellungen",
+  "Invalid address": "Ungültige Adresse",
+  "Invalid dates": "Ungültiges Datum",
+  "MMM DD YYYY": "MMM DD YYYY",
+  "MMM DD YYYY hh:mm": "MMM DD YYYY hh:mm",
+  "MMM DD YYYY hh:mm:ss": "MMM DD YYYY hh:mm:ss",
+  "MMM DD YYYY hh[h]": "MMM DD YYYY hh[h]",
+  "MMM YYYY": "MMM YYYY",
+  "Passphrase:": "Passphrase",
+  "Removed votes": "Entfernte Stimmen",
+  "YYYY": "YYYY",
+  "You don’t have enough balance to enable it. {{fee}} LSK is required.": "You don’t have enough balance to enable it. {{fee}} LSK is required.",
+  "Your {{walletModel}} has an old Firmware. Please update it.": "Your {{walletModel}} has an old Firmware. Please update it.",
+  "[Today], hh:mm A": "[Today], hh:mm A",
```

Comment:
```suggestion
  "[Today], hh:mm A": "[Heute], HH:mm",
```
Else it's
![Screenshot 2019-09-02 at 15 48 07](https://user-images.githubusercontent.com/1254342/64119012-22db5280-cd99-11e9-9554-be4b01d7481e.png)

Similar for the two lines below. We need this because the requirement was to use a custom date format (that we pass to moment.js to override its default) for English. So we need to define our own custom date format for German as well.

Reply:
I don't understand why these are not done by momentjs? It takes care of relative time generation.